pkg/client/fake/client_test.go

```diff
@@ -726,6 +726,54 @@ var _ = Describe("Fake client", func() {
 			Expect(apierrors.IsNotFound(err)).To(BeTrue())
```

Comment:
Any reason file permissions have changed?

Reply:
Could you clarify what you mean by this? My understanding was that by removing the object's finalizer, the resource is garbage collected, resulting in a 404 error when calling Get()

Reply:
@vincepri Could you clarify what you mean here? My understanding is that `Expect(apierrors.IsNotFound(err)).To(BeTrue())` is a check to ensure that once the object's finalizer is removed, it is garbage collected, and a call to `Get()` returns a 404 error.

Reply:
The diff was showing the file (client_test.go) permissions changed (to 777), it can be resolved now, seems ok

Reply:
Ah, I see. It's currently 755, should I change it back to 644?